Doc/whatsnew/3.9.rst

```diff
@@ -912,6 +912,12 @@ Removed
   unicode characters.
 
 
+Platform Support Removals
```

Comment:
Support for AIX 5.3 and below was removed in Python 3.9 when bpo-30680 was not fixed. Do you not issue errata?

Reply:
If you want to update the What's New in Python 3.9 document, please write a separated PR. This PR cannot be backported to the 3.9 branch since it contains other changes.